hat/conf/dev.conf

```diff
@@ -8,6 +8,23 @@ play {
   }
 }
 
+
```

Comment:
Should this move into separate branch and be added with the overall improvements of the dev environment?